mathesar_ui/src/stores/table-data/tabularData.ts

```diff
@@ -39,9 +39,19 @@ export interface TabularDataProps {
   hasEnhancedPrimaryKeyCell?: Parameters<
     typeof processColumn
   >[0]['hasEnhancedPrimaryKeyCell'];
+  /**
+   * When true, load the record summaries associated directly with each record
+   * in the table. These are *not* the record summaries associated with linked
+   * records. Instead they are the summaries of the records themselves. By
+   * default, we don't load these summaries because it's a performance hit. But
+   * we need them for the records within the record selector.
+   */
+  loadIntrinsicRecordSummaries?: boolean;
```

Comment:
@seancolsen The only place we seem to require intrinsic record summaries is when selecting a record via the record selector which updates a cell in the parent table. However, the RPC method `records.patch` already returns the required record summary within `linked_record_summaries`. Could we use that directly instead of requesting the intrinsic record summaries?

It is possible that I may not be seeing additional places/cases where this is required. If so, could you specify where/how we need them?

Reply:
Ah.. I see that we close the Record selector immediately after picking the record, and the summary is required while the parent record is being saved.